victron_mqtt/_victron_topics.py

```diff
@@ -509,6 +509,17 @@
         device_type=DeviceType.BATTERY,
         value_type=ValueType.INT,
     ),
+    TopicDescriptor(
+        topic="N/+/battery/+/Info/ChargeMode",
+        message_type=MetricKind.ATTRIBUTE,
+        short_id="charge_mode",
+        name="Battery charge mode",
+        device_type=DeviceType.BATTERY,
+        unit_of_measurement=None,
+        metric_type=MetricType.NONE,
+        metric_nature=MetricNature.NONE,
+        value_type=ValueType.STRING,
```

Comment:
The value is probably not a string but an enum with multiple options. 
As I dont find it [here](https://github.com/victronenergy/venus/wiki/dbus#battery), I'm not sure how to proceed. Do you have any docuemtnation about this one?